spec/helpers/searches/paths_helper_spec.rb

```diff
@@ -1,22 +1,28 @@
 require 'spec_helper'
 
 describe Searches::PathsHelper do
-
   fixtures :affiliates, :i14y_drawers, :i14y_memberships
   let(:affiliate) { affiliates(:power_affiliate) }
 
-  describe '#path_for_filterable_search' do
+  describe '#path_for_filterable_search', type: :request do
```

Comment:
@nickmarden , it feels a little dirty to make this an integration test, but I wasn't able to find a way to stub the current path used by the `url_for` method (I barked up a number of trees before finding this solution). If you have such a way, I'm all ears.